### PR TITLE
[ID] Add `Unlock Requirements`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**

### CODE REVIEW

**Comment by TicClick** [comment]
please apply #6109 to your translation

File: wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/id.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Persyaratan untuk mendapatkan medali Beatmap Challenge Packs
+
+| Nama medali | Persyaratan |
+| :-- | :-- |
+| Mappers' Guild Pack VII | Selesaikan tingkat kesulitan tersusah seluruh beatmap yang termasuk dalam beatmap pack [Mappers' Guild Pack VII](https://osu.ppy.sh/beatmaps/packs/2034). Seluruh beatmap pada pack ini memiliki tingkat kesulitan tersusah pada jangkauan star rating 5.00★–5.99★. |
```
Comment (mousesalt) [inline wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/id.md:5]:
```suggestion
| Mappers' Guild Pack VII | Selesaikan tingkat kesulitan yang paling susah dari seluruh beatmap, termasuk dalam beatmap pack [Mappers' Guild Pack VII](https://osu.ppy.sh/beatmaps/packs/2034). Seluruh beatmap pada pack ini memiliki tingkat kesulitan tersusah pada jangkauan star rating 5.00★–5.99★. |
```

Comment (mousesalt) [reply wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/id.md:5]: hardest is the top of harder

File: wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/id.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+| Mappers' Guild Pack VIII | Selesaikan tingkat kesulitan tersusah seluruh beatmap yang termasuk dalam beatmap pack [Mappers' Guild Pack VIII](https://osu.ppy.sh/beatmaps/packs/2035). Seluruh beatmap pada pack ini memiliki tingkat kesulitan tersusah pada jangkauan star rating 6.00★–6.99★. |
```
Comment (mousesalt) [inline wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/id.md:6]:
```suggestion
| Mappers' Guild Pack VIII | Selesaikan tingkat kesulitan yang paling susah dari seluruh beatmap, termasuk dalam beatmap pack [Mappers' Guild Pack VIII](https://osu.ppy.sh/beatmaps/packs/2035). Seluruh beatmap pada pack ini memiliki tingkat kesulitan tersusah pada jangkauan star rating 6.00★–6.99★. |
```

File: wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/id.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+| Mappers' Guild Pack IX | Selesaikan tingkat kesulitan tersusah seluruh beatmap yang termasuk dalam beatmap pack [Mappers' Guild Pack IX](https://osu.ppy.sh/beatmaps/packs/2036). Seluruh beatmap pada pack ini memiliki tingkat kesulitan tersusah pada jangkauan star rating 7.00★–7.99★. |
```
Comment (mousesalt) [inline wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/id.md:7]:
```suggestion
| Mappers' Guild Pack IX | Selesaikan tingkat kesulitan yang paling susah dari seluruh beatmap, termasuk dalam beatmap pack [Mappers' Guild Pack IX](https://osu.ppy.sh/beatmaps/packs/2036). Seluruh beatmap pada pack ini memiliki tingkat kesulitan tersusah pada jangkauan star rating 7.00★–7.99★. |
```

File: wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/id.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+| Camellia II | Selesaikan seluruh beatmap yang termasuk dalam beatmap pack [Camellia Challenges Pack](https://osu.ppy.sh/beatmaps/packs/2053). Seluruh beatmpa pada pack ini memiliki tingkat kesulitan yang diklasifikasikan sebagai tingkat kesulitan `Expert` atau lebih tinggi. |
```
Comment (mousesalt) [inline wiki/Medals/Unlock_requirements/Beatmap_challenge_packs/id.md:8]:
```suggestion
| Camellia II | Selesaikan seluruh beatmap yang termasuk dalam beatmap pack [Camellia Challenges Pack](https://osu.ppy.sh/beatmaps/packs/2053). Seluruh beatmap pada pack ini memiliki tingkat kesulitan yang diklasifikasikan sebagai tingkat kesulitan `Expert` atau lebih tinggi. |
```

File: wiki/Medals/Unlock_requirements/Dedication/id.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# Persyaratan untuk mendapatkan medali Dedication
+
+## osu!
+
+| Nama medali | Persyaratan | Deskripsi |
+| :-- | :-- | :-- |
+| 5,000 Plays | Raih jumlah main sebanyak 5.000 kali bermain pada mode permainan osu!. | There's a lot more where that came from. |
```
Comment (mousesalt) [inline wiki/Medals/Unlock_requirements/Dedication/id.md:7]: `main` to `bermain`

File: wiki/Medals/Unlock_requirements/Dedication/id.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+| 15,000 Plays | Raih jumlah main sebanyak 15.000 kali bermain pada mode permainan osu!. | Must.. click.. circles.. |
+| 25,000 Plays | Raih jumlah main sebanyak 25.000 kali bermain pada mode permainan osu!. | There's no going back. |
+| 50,000 Plays | Raih jumlah main sebanyak 50.000 kali bermain pada mode permainan osu!. | You're here forever. |
+
+## osu!taiko
+
+| Nama medali | Persyaratan | Deskripsi |
+| :-- | :-- | :-- |
+| 30,000 Drum Hits | Raih jumlah akumulatif pukulan drum sebanyak 30.000 pukulan pada mode permainan osu!taiko. | Did that drum have a face? |
```
Comment (mousesalt) [inline wiki/Medals/Unlock_requirements/Dedication/id.md:16]: `akumulatif` and `total` is standing to one, suggest to just delete `akumulatif`

File: wiki/Medals/Unlock_requirements/Skill/id.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# Persyaratan untuk mendapatkan medali Skill
+
+| Nama medali | Persyaratan |
+| :-- | :-- |
+| 500 Combo | Capai 500 combo pada sebuah beatmap dengan mode permainan apapun. |
```
Comment (mousesalt) [inline wiki/Medals/Unlock_requirements/Skill/id.md:5]: `combo` to `kombo`

Comment (ZeroPyrozen) [reply wiki/Medals/Unlock_requirements/Skill/id.md:5]: I'm keeping the `combo` and `full combo` to sync with [\[ID\] Combo](https://github.com/ppy/osu-wiki/blob/master/wiki/Glossary/Combo_(score_multiplier)/id.md) and [\[ID\] Full Combo](https://github.com/ppy/osu-wiki/blob/master/wiki/Full_combo/id.md) articles

File: wiki/Medals/Unlock_requirements/Dedication/id.md (hunk 6 above)
Comment (ZeroPyrozen) [reply wiki/Medals/Unlock_requirements/Dedication/id.md:16]: I was really confused how to tell the drum hit, note key, and caught fruit count achievement doesn't have to be achieved in single play.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**